modules/machine-user-provisioned-limitations.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * machine_management/applying_autoscaling.adoc
+// * machine_management/creating-infrastructure-machinesets.adoc
+// * machine_management/creating-machinesets.adoc
+// * machine_management/deploying-machine-health-checks.adoc
+// * machine_management/manually-scaling-machinesets.adoc
+
+[IMPORTANT]
+====
+This process is not applicable to clusters that use user-provisioned
```

Comment:
nothing would stop you from running machine scalable resources in e.g upi aws if you configure it properly

Reply:
@enxebre, that's been a contentious point during reviews before GA. Do you think that it's time to document that path?